lms/djangoapps/dashboard/sysadmin.py

```diff
@@ -40,6 +40,7 @@
 from openedx.core.djangoapps.external_auth.views import generate_password
 from student.models import CourseEnrollment, Registration, UserProfile
 from student.roles import CourseInstructorRole, CourseStaffRole
+from util.milestones_helpers import remove_course_references
```

Comment:
Please reorder imports.

Reply:
Sorry, I've got stuck trying to understand where should I place it. Could you explain?

Reply:
Sorry about misleading, with order absence in the import block you can put your lib wherever you want.